script/core.hbs

```diff
@@ -7,6 +7,16 @@
 <script src="https://assets.sitescdn.net/answers/v{{global_config.sdkVersion}}/{{#if params.sdkLocaleOverride}}{{#unless (eq params.sdkLocaleOverride 'en')}}{{params.sdkLocaleOverride}}-{{/unless}}{{else if global_config.locale}}{{#unless (eq global_config.locale 'en')}}{{global_config.locale}}-{{/unless}}{{/if}}answerstemplates.compiled.min.js" defer></script>
 <script>
 {{#babel}}
+  function onAnswersScriptLoad () {
+    const waitBeforeInitPromises = [
```

Comment:
I think we should come up with a more descriptive name here. Maybe `preAnswersInitTasks` or something like that?

Reply:
ya, I like that better